editions/tw5.com/tiddlers/gettingstarted/GettingStarted.tid

```diff
@@ -15,7 +15,7 @@ Brief instructions for getting started on the different platforms and configurat
 
 Troubleshooting:
 
-* Don't attempt to use the browser ''File''/''Save'' menu option to save changes (it doesn't work)
+* Don't attempt to use the browser ''File''/''Save'' menu option to save changes (it doesn't work).
```

Comment:
Yup, the house style is to omit the trailing full stop in list items. There is a note about it at http://tiddlywiki.com/#Typography:

> Items in lists and tables should only end with a full stop (period in US English) if they are complete sentences. They should have no trailing punctuation otherwise.